tests/range/jsfmt.spec.js

```diff
@@ -1 +1,18 @@
-run_spec(__dirname, { rangeStart: 180, rangeEnd: 220 });
+run_file(__dirname, "range.js", { rangeStart: 224, rangeEnd: 227 });
+run_file(__dirname, "try-catch.js", { rangeStart: 7, rangeEnd: 8 });
+run_file(__dirname, "ignore-indentation.js", {
+  rangeStart: 222,
+  rangeEnd: 225
```

Comment:
Yeah, that was getting annoying, especially since it wasn't obvious where the range was. I decided to go with `<<<PRETTIER_RANGE_START>>>` and `<<<PRETTIER_RANGE_END>>>` to make it more visually obvious, more like invalid JS code, and to avoid conflicts with any similar future changes for https://github.com/prettier/prettier/pull/1637. See https://github.com/prettier/prettier/pull/1659/commits/c5aa2e38380e4fc7b2c63ea8a1f3793dfcd39a15

I also removed the `run_file` function, since we don't need it anymore: https://github.com/prettier/prettier/pull/1659/commits/64fa6659cb602cd765732c6d08d2a646f53b76cc